src/tests/unit/services/F2fService.test.ts

```diff
@@ -250,14 +251,14 @@ describe("F2f Service", () => {
 			}],
 		});
 
-		const result = await f2fService.getSessionsByAuthSessionState("F2F_SESSION_STARTED");
+		const result = await f2fService.getSessionsByAuthSessionStates(["F2F_YOTI_SESSION_CREATED, F2F_AUTH_CODE_ISSUED, F2F_ACCESS_TOKEN_ISSUED"]);
```

Comment:
Wait - this is an array with one string in it is that what we're expecting? or should each state be it's own string?

Reply:
And then we have an `expect(mockDynamoDbClient.query..` for each of those?

Reply:
https://github.com/alphagov/di-ipv-cri-f2f-api/pull/252/commits/2f01547dd32d70e5564cea351d6d14d11e1cbb5f